rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -450,6 +524,7 @@
     function renderAll() {
       renderControls();
       renderChartContainer();
+      setTimeout(renderMathHJax, 100);
```

Comment:
One question to start out: why do we need this timeout, and is there any particular reason you chose 100ms? My instinct is that we should handle this special math notation rendering as part of the logic for rendering a specific chart (maybe as part of `renderChart`).

Reply:
changed logic, followed example to make it work https://stackoverflow.com/questions/37359888/mathjax-not-rendering-inline-labels